amd_openvx_extensions/amd_rpp/source/tensor/Downmix.cpp

```diff
@@ -0,0 +1,222 @@
+/*
+Copyright (c) 2024 Advanced Micro Devices, Inc. All rights reserved.
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+*/
+
+#include "internal_publishKernels.h"
+#include "vx_ext_amd.h"
+
+struct DownmixLocalData
+{
+    vxRppHandle *handle;
+    Rpp32u device_type;
+    RppPtr_t pSrc;
+    RppPtr_t pDst;
+    vx_int32 *pSamples;
+    vx_int32 *pChannels;
+    RpptDescPtr pSrcDesc;
+    RpptDescPtr pDstDesc;
+    RpptDesc srcDesc;
+    RpptDesc dstDesc;
```

Comment:
The srcDesc and dstDesc are not needed please check the Brightness.cpp file as to how it is done

Reply:
This is not addressed please check